app/assets/stylesheets/base/_global.scss

```diff
@@ -1,9 +1,32 @@
+header {
+  background-image: (url("puppies4.jpg"));
```

Comment:
@Tallys This should have a fallback color in case the user has images turned off.